code/game/gamemodes/malfunction/malfunction.dm

```diff
@@ -142,6 +142,12 @@ You should now be able to use your Explode spell to interface with the nuclear f
 			all_dead = 0
 	return all_dead
 
+/datum/game_mode/proc/can_malf_ai_takeover()
+	for(var/datum/mind/AI_mind in malf_ai) //if there happens to be more than one malfunctioning AI, there only needs to be one in the main station: the crew can just kill that one and the countdown stops while they get the rest
+		var/turf/T = get_turf(AI_mind.current)
+		if(T && (T.z == map.zMainStation))
```

Comment:
I'm with Bauds here.
Sometimes redundant parentheses should be removed, but this isn't one of those times.